sdk/python/tests/integration/online_store/test_universal_online.py

```diff
@@ -490,7 +490,7 @@ def test_online_retrieval(
     )
 
     entity_sample = datasets.orders_df.sample(10)[
-        ["customer_id", "driver_id", "order_id", "event_timestamp"]
+        ["customer_id", "driver_id", "order_id", "origin_id", "event_timestamp"]
```

Comment:
is this relevant to this pr or is it just a datasource that we forgot to include in the test.

Reply:
this is relevant since we now want to retrieve features from the pushable_location_stats SFV, which requires a new join key (specifically a location_id), so when we sample, we also need to pull a location_id (in this case the column has been renamed to origin_id, so we pull that column)

Reply:
Ah sounds good.